## 26% of end-end tests have passed
```
Total executed tests: 43
Failed tests: 32
Passed tests: 11
```
```
IDs of failed tests: 702845,702840,702894,703086,702855,702947,702958,702786,702745,702731,702844,703202,702841,703496,702807,702839,702733,702808,702809,702838,702813,702948,702859,702957,702783,702843,703391,702732,703503,702936,703495,703194 
```
<h3>Failed tests (32)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&PR_ID=17042&TR_CASE_IDS=702845,702840,702894,703086,702855,702947,702958,702786,702745,702731,702844,703202,702841,703496,702807,702839,702733,702808,702809,702838,702813,702948,702859,702957,702783,702843,703391,702732,703503,702936,703495,703194">Rerun failed tests</a></li><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224528">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `Button` by `accessibility id`: `show-request-to-join-screen-button` </div></blockquote></p>

```
critical/test_public_chat_browsing.py:338: in test_community_discovery
    if element.is_element_differs_from_template(template):
../views/base_element.py:278: in is_element_differs_from_template
    difference = ImageChops.difference(self.image, self.template)
../views/base_element.py:251: in image
    return Image.open(BytesIO(base64.b64decode(self.find_element().screenshot_as_base64)))
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2c4356cc496243acaec428375f326ca0?auth=e1f757df2e0ef9e78e283d696617c444#54">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224515">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td>

```
Test setup failed: medium/test_activity_center.py:176: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224518">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: medium/test_activity_center.py:176: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224517">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td>

```
Test setup failed: medium/test_activity_center.py:176: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224514">test_navigation_jump_to</a>, id: 702936 </b></td></tr><tr><td><p><blockquote><div>Device 2: Joining community </div><div>Device 2: Find `Button` by `accessibility id`: `show-request-to-join-screen-button` </div></blockquote></p>

```
Test setup failed: medium/test_activity_center.py:176: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/3a7c179df618439f9836d87bbd464328?auth=adda17cb89f6ae61722b8801eb01e5eb">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f86831b13f734bf289acdff197d3aaba?auth=33b23cb780e6f62fd3fde0280cb4452c">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224522">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `authors-for-reaction-5` </div><div>Device 1: Tap on found: Button </div></blockquote></p>

```
critical/chats/test_group_chat.py:344: in test_group_chat_reactions
    self.errors.verify_no_errors()
base_test_case.py:187: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Incorrect users are shown for 'laugh' reaction.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4f706a9c2b3d44b391f7181f99772f04?auth=eb0c176e9d5fd84ca37639a9a0280220#158">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfbb800fd88a43a89fe0ded5dfea71b9?auth=20dfc0badb2f2ace782dfc45554d7c54#98">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fa68c1ae036741c7abe971db47bb1525?auth=de9bb8f37047ff133fc63d13c4f81458#78">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224496">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><p><blockquote><div>Device 2: `Text` is `Delivered` </div><div>Device 1: Looking for a message by text: Hey, admin! </div></blockquote></p>

```
critical/chats/test_group_chat.py:231: in test_group_chat_join_send_text_messages_push
    self.errors.verify_no_errors()
base_test_case.py:187: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 No PN was received on new message for message in group chat
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4f706a9c2b3d44b391f7181f99772f04?auth=eb0c176e9d5fd84ca37639a9a0280220#125">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfbb800fd88a43a89fe0ded5dfea71b9?auth=20dfc0badb2f2ace782dfc45554d7c54#71">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fa68c1ae036741c7abe971db47bb1525?auth=de9bb8f37047ff133fc63d13c4f81458#71">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224497">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><p><blockquote><div>Device 3: Looking for a message by text: message from old member </div><div>Device 3: Looking for a message by text: message from new member </div></blockquote></p>

```
critical/chats/test_group_chat.py:441: in test_group_chat_offline_pn
    self.errors.verify_no_errors()
base_test_case.py:187: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Messages PN was not fetched from offline
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4f706a9c2b3d44b391f7181f99772f04?auth=eb0c176e9d5fd84ca37639a9a0280220#304">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfbb800fd88a43a89fe0ded5dfea71b9?auth=20dfc0badb2f2ace782dfc45554d7c54#199">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fa68c1ae036741c7abe971db47bb1525?auth=de9bb8f37047ff133fc63d13c4f81458#177">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224488">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224526">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div>Device 3: Sending message 'Chat is unmuted now' </div><div>Device 3: Find `ChatMessageInput` by `accessibility id`: `chat-message-input` </div></blockquote></p>

```
critical/chats/test_group_chat.py:587: in test_group_chat_mute_chat
    self.chats[2].send_message(unmuted_message)
../views/chat_view.py:989: in send_message
    self.chat_message_input.send_keys(message)
../views/base_element.py:375: in send_keys
    self.find_element().send_keys(value)
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/webelement.py:235: in send_keys
    self._execute(RemoteCommand.SEND_KEYS_TO_ELEMENT, {'text': ''.join(keys), 'value': keys})
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
/home/jenkins/.local/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
/home/jenkins/.local/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 androidx.test.uiautomator.StaleObjectException; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: androidx.test.uiautomator.StaleObjectException
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:63)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:919)
E   Caused by: androidx.test.uiautomator.StaleObjectException
E   	at androidx.test.uiautomator.UiObject2.getAccessibilityNodeInfo(UiObject2.java:647)
E   	at androidx.test.uiautomator.UiObject2.hashCode(UiObject2.java:105)
E   	at java.lang.Object.toString(Object.java:291)
E   	at java.util.Formatter$FormatSpecifier.printString(Formatter.java:2978)
E   	at java.util.Formatter$FormatSpecifier.print(Formatter.java:2855)
E   	at java.util.Formatter.format(Formatter.java:2524)
E   	at java.util.Formatter.format(Formatter.java:2459)
E   	at java.lang.String.format(String.java:2870)
E   	at io.appium.uiautomator2.utils.ReflectionUtils.invoke(ReflectionUtils.java:85)
E   	at io.appium.uiautomator2.core.AxNodeInfoExtractor.extractAxNodeInfo(AxNodeInfoExtractor.java:58)
E   	at io.appium.uiautomator2.core.AxNodeInfoExtractor.toAxNodeInfo(AxNodeInfoExtractor.java:48)
E   	at io.appium.uiautomator2.utils.ElementHelpers.setText(ElementHelpers.java:92)
E   	at io.appium.uiautomator2.model.BaseElement.setText(BaseElement.java:156)
E   	at io.appium.uiautomator2.handler.SendKeysToElement.setText(SendKeysToElement.java:86)
E   	at io.appium.uiautomator2.handler.SendKeysToElement.safeHandle(SendKeysToElement.java:115)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	... 33 more
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4f706a9c2b3d44b391f7181f99772f04?auth=eb0c176e9d5fd84ca37639a9a0280220#355">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfbb800fd88a43a89fe0ded5dfea71b9?auth=20dfc0badb2f2ace782dfc45554d7c54#224">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fa68c1ae036741c7abe971db47bb1525?auth=de9bb8f37047ff133fc63d13c4f81458#216">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224510">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 1: Find `Button` by `accessibility id`: `jump-to` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1185: in test_1_1_chat_edit_message
    self.chat_1.jump_to_card_by_text(self.username_2)
../views/base_view.py:638: in jump_to_card_by_text
    self.click_on_floating_jump_to()
../views/base_view.py:627: in click_on_floating_jump_to
    self.jump_to_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `jump-to` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2e5996655da244f98046cc2e2d1091e7?auth=5fa7282a19495560e5bfeb2352c4ace1#286">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cd92aaba0b546f6af6c8eac8580dd16?auth=cfa3fdb6abed6fbe7741cc3844ac6b87#214">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224491">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `MemberPhoto` by `xpath`: `//*[starts-with(@text,'profile_photo')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']//*[@content-desc='user-avatar']` </div><div>Device 2: Image differs from template to 6.745557598039216 percents </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1127: in test_1_1_chat_non_latin_messages_stack_update_profile_photo
    self.errors.verify_no_errors()
base_test_case.py:187: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Image of user in 1-1 chat is too different from template!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2e5996655da244f98046cc2e2d1091e7?auth=5fa7282a19495560e5bfeb2352c4ace1#192">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cd92aaba0b546f6af6c8eac8580dd16?auth=cfa3fdb6abed6fbe7741cc3844ac6b87#149">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224487">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td>

```
Test is not run, e2e blocker  
```

<b>[[reason: [NOTRUN] Pin feature is in development]]</b><br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224527">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><p><blockquote><div>Device 1: Could not reach home view by pressing system back button </div><div>Device 1: Find `ChatsTab` by `accessibility id`: `chats-stack-tab` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1382: in test_1_1_chat_mute_chat
    self.home_1.chats_tab.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: ChatsTab by accessibility id: `chats-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2e5996655da244f98046cc2e2d1091e7?auth=5fa7282a19495560e5bfeb2352c4ace1#331">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cd92aaba0b546f6af6c8eac8580dd16?auth=cfa3fdb6abed6fbe7741cc3844ac6b87#249">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224489">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `jump-to` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1273: in test_1_1_chat_text_message_delete_push_disappear
    self.chat_1.jump_to_card_by_text(self.username_2)
../views/base_view.py:638: in jump_to_card_by_text
    self.click_on_floating_jump_to()
../views/base_view.py:627: in click_on_floating_jump_to
    self.jump_to_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `jump-to` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2e5996655da244f98046cc2e2d1091e7?auth=5fa7282a19495560e5bfeb2352c4ace1#316">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cd92aaba0b546f6af6c8eac8580dd16?auth=cfa3fdb6abed6fbe7741cc3844ac6b87#234">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224499">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `ProfileButton` by `accessibility id`: `open-profile` </div><div>Device 1: Find `ChatsTab` by `accessibility id`: `chats-stack-tab` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1135: in test_1_1_chat_push_emoji
    self.home_1.chats_tab.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: ChatsTab by accessibility id: `chats-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2e5996655da244f98046cc2e2d1091e7?auth=5fa7282a19495560e5bfeb2352c4ace1#261">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cd92aaba0b546f6af6c8eac8580dd16?auth=cfa3fdb6abed6fbe7741cc3844ac6b87#197">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224493">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 1: Find `Button` by `accessibility id`: `jump-to` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1332: in test_1_1_chat_is_shown_message_sent_delivered_from_offline
    self.chat_1.jump_to_card_by_text(self.username_2)
../views/base_view.py:638: in jump_to_card_by_text
    self.click_on_floating_jump_to()
../views/base_view.py:627: in click_on_floating_jump_to
    self.jump_to_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `jump-to` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2e5996655da244f98046cc2e2d1091e7?auth=5fa7282a19495560e5bfeb2352c4ace1#324">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cd92aaba0b546f6af6c8eac8580dd16?auth=cfa3fdb6abed6fbe7741cc3844ac6b87#239">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224525">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `accessibility id`: `jump-to` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:1212: in test_1_1_chat_send_image_save_and_share
    self.chat_1.jump_to_card_by_text(self.username_2)
../views/base_view.py:638: in jump_to_card_by_text
    self.click_on_floating_jump_to()
../views/base_view.py:627: in click_on_floating_jump_to
    self.jump_to_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `jump-to` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2e5996655da244f98046cc2e2d1091e7?auth=5fa7282a19495560e5bfeb2352c4ace1#308">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cd92aaba0b546f6af6c8eac8580dd16?auth=cfa3fdb6abed6fbe7741cc3844ac6b87#229">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224506">test_community_leave</a>, id: 702845 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224502">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224513">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224519">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224495">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception 
```

<b>[[Issue with username in PN, issue #6 in https://github.com/status-im/status-mobile/issues/15500]]</b><br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224505">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224503">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224501">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224498">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224500">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div>Device 2: Joining community </div><div>Device 2: Find `Button` by `accessibility id`: `show-request-to-join-screen-button` </div></blockquote></p>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/be3c3fb9da6744f28d9e216f3dd4e447?auth=3924e07bd275b8e7991854289cf35405">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dca1654ad2e34cd4977c97259fe80788?auth=74227e82459016f89fa9453bd325d609">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224516">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224511">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>13. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224504">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>14. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224521">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td>

```
Test setup failed: critical/test_public_chat_browsing.py:577: in prepare_devices
    self.community_2.join_community()
../views/chat_view.py:415: in join_community
    self.join_button.click()
../views/base_element.py:90: in click
    self.find_element().click()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 2: Button by accessibility id: `show-request-to-join-screen-button` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table></details><h3>Passed tests (11)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224509">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4ee0b49704d942dd941179547c181276?auth=2c63d6eee5b78decbd4c9ce8daf69929#92">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6268019e7ffd4f35a3250c8d44dc5d2d?auth=5a616c7c2f276021acacb532ca818321#88">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224508">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4ee0b49704d942dd941179547c181276?auth=2c63d6eee5b78decbd4c9ce8daf69929#55">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6268019e7ffd4f35a3250c8d44dc5d2d?auth=5a616c7c2f276021acacb532ca818321#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224486">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2e5996655da244f98046cc2e2d1091e7?auth=5fa7282a19495560e5bfeb2352c4ace1#78">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cd92aaba0b546f6af6c8eac8580dd16?auth=cfa3fdb6abed6fbe7741cc3844ac6b87#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224492">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2e5996655da244f98046cc2e2d1091e7?auth=5fa7282a19495560e5bfeb2352c4ace1#130">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cd92aaba0b546f6af6c8eac8580dd16?auth=cfa3fdb6abed6fbe7741cc3844ac6b87#108">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224494">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2e5996655da244f98046cc2e2d1091e7?auth=5fa7282a19495560e5bfeb2352c4ace1#350">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6cd92aaba0b546f6af6c8eac8580dd16?auth=cfa3fdb6abed6fbe7741cc3844ac6b87#252">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224523">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4f706a9c2b3d44b391f7181f99772f04?auth=eb0c176e9d5fd84ca37639a9a0280220#261">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/cfbb800fd88a43a89fe0ded5dfea71b9?auth=20dfc0badb2f2ace782dfc45554d7c54#162">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/fa68c1ae036741c7abe971db47bb1525?auth=de9bb8f37047ff133fc63d13c4f81458#130">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224490">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2c4356cc496243acaec428375f326ca0?auth=e1f757df2e0ef9e78e283d696617c444#120">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224507">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2c4356cc496243acaec428375f326ca0?auth=e1f757df2e0ef9e78e283d696617c444#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224520">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2c4356cc496243acaec428375f326ca0?auth=e1f757df2e0ef9e78e283d696617c444#271">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224512">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2c4356cc496243acaec428375f326ca0?auth=e1f757df2e0ef9e78e283d696617c444#145">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3224524">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2c4356cc496243acaec428375f326ca0?auth=e1f757df2e0ef9e78e283d696617c444#164">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/integration_test.cljs
+++ b/src/status_im/integration_test.cljs
@@ -231,7 +231,7 @@
     [compressed-key "zQ3shWj4WaBdf2zYKCkXe6PHxDxNTzZyid1i75879Ue9cX9gA"
      public-key
      "0x048a6773339d11ccf5fd81677b7e54daeec544a1287bd92b725047ad6faa9a9b9f8ea86ed5a226d2a994f5f46d0b43321fd8de7b7997a166e67905c8c73cd37cea"
-     three-words-name "Rich Total Pondskater"]
+     primary-name "zQ3...9cX9gA"]
     (rf-test/run-test-async
      (initialize-app!)
      (rf-test/wait-for
@@ -254,6 +254,6 @@
              (rf-test/wait-for
                [:contacts/contact-built]
                (let [contact @(rf/subscribe [:contacts/current-contact])]
-                 (is (= three-words-name (:primary-name contact))))
+                 (is (= primary-name (:primary-name contact))))
                (logout!)
                (rf-test/wait-for [::logout/logout-method])))))))))

--- a/status-go-version.json
+++ b/status-go-version.json
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.166.3",
-    "commit-sha1": "347d875acb290929e93969bee2b2c178d4c50ac2",
-    "src-sha256": "036vgywbd10pwddam2q9f7sxfan2rbaj8l9khph7wxn2l37476q1"
+    "version": "v0.166.4",
+    "commit-sha1": "bf748de216a2061bf14afb2a410599485ee1ee7c",
+    "src-sha256": "0acpdc8sw6i6zpxyfzahrfbr270ldcf3jv8g4bp4kh59q2mf6bxz"
 }

